src/Utility/InputValidator.php

```diff
@@ -109,4 +109,27 @@ public static function is_curl_handle($input) {
 
 		return false;
 	}
+
+	/**
+	 * Verify that a received input parameter is a valid "token name" according to the
+	 * specification in RFC 2616 (HTTP/1.1).
+	 *
+	 * The short version is: 1 or more ascii characters, CTRL chars and separators not allowed.
```

Comment:
I'd capitalize the ASCII acronym, it is not an actual word.
```suggestion
	 * The short version is: 1 or more ASCII characters, CTRL chars and separators not allowed.
```